c/polyjuice.h

```diff
@@ -1387,6 +1443,30 @@ int run_polyjuice() {
     return ret;
   }
 
+  /**
+   * Only handle EOA transfer, if
+   * - to_id is g_creator_account_id
+   * - not create contract
+   * - g_eoa_transfer_flag 
+   * - g_eoa_transfer_to_address
```

Comment:
We should also check that to address is not a contract.
If to address is a contract, we should throw error. Because to contract will not have knowledge of this transfer, users will lock and lose their pCKB.

Reply:
see https://github.com/nervosnetwork/godwoken-polyjuice/pull/173/files#diff-8a43ccc334a831d56b5bdfa4712228904f3823f6ec2b82941664ca849ab496b0R1068

Reply:
1. if to is contract address ,will invoke `fallback` function ,it will successful that contract contains `fallback` or `receive`  function and contains `payable` tag 
2. and it should be successful that  to is `precompile conract` or `0x`  
to is 0x1: https://rinkeby.etherscan.io/tx/0x9813871dec4d4eb9ae5df084adbad967e276674fc45fcad0b4375e0e52565e59
to is 0x0:https://rinkeby.etherscan.io/tx/0x992e75ad5dbc2b3092ace49d12c95fefb84c3da7f731dea77b71effab797dbbf

Reply:
That is correct. It's tricky to understand the difference between `to_id` and `to_address` here.
In polyjuice, we always use the CREATOR_ID, defined in config, as `to_id`, if this tx is about creating a contract or transferring native tokens (in this case). When we transfer native tokens, we must set the **real** `to_address` in args. And this `to_address` cannot be a contract address.
In this way, we will not enter into EVM! We separate two branches, one for eoa transferring, the other one for the rest.

In your case, the `to_id` is not CREATOR_ID. And it will enter into EVM just like the old days.